.reaction/jsdoc/templates/static/README.md

```diff
@@ -0,0 +1,32 @@
+# Reaction Commerce API Docs
+
+## API Docs
+
+Start here:
+
+- [Reaction Components API](Components.html)
+- [Reaction Core schemas](schemas.html)
```

Comment:
Why is the casing different between `Components.html` and `schemas.html`?

Reply:
Because of the way the namespace was originally cased:
http://api.docs.reactioncommerce.com/schemas.html
http://api.docs.reactioncommerce.com/Components.html

I could capitalize Schemas in a different PR.